doc/book/admin/start_stop_instance.rst

```diff
@@ -1,214 +1,335 @@
+    ..  code-block:: yaml
 
-    $ tarantoolctl start my_app
+        storage1:
+        storage2:
+        router:
 
-Expect to see messages indicating that the instance has started. Then:
+*   ``init.lua`` -- the code of ``storage1`` and ``storage2``
+*   ``router.init.lua`` -- the code of ``router``
 
-.. code-block:: console
 
-    $ ls -l ~/tarantool_test/my_app
+Identifying instances in code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the application is working, each instance has associated environment variables
+``TARANTOOL_INSTANCE_NAME`` and ``TARANTOOL_APP_NAME``. You can use them in the application
+code to identify the instance on which the code runs.
+
+To obtain the instance and application names, use the following code:
+
+..  code:: lua
+
+    local inst_name = os.getenv('TARANTOOL_INSTANCE_NAME')
+    local app_name = os.getenv('TARANTOOL_APP_NAME')
+
+
+Managing multi-instance applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start all three instances of the ``demo`` application:
+
+..  code-block:: console
+
+    $ tt start demo
+       • Starting an instance [demo:router]...
+       • Starting an instance [demo:storage1]...
+       • Starting an instance [demo:storage2]...
+
+Check the status of ``demo`` instances:
+
+..  code-block:: console
+
+    $ tt status demo
+    INSTANCE         STATUS      PID
+    demo:router      RUNNING     55
+    demo:storage1    RUNNING     56
+    demo:storage2    RUNNING     57
+
+Check the status of a specific instance:
+
+..  code-block:: console
+
+    $ tt status demo:router
+    INSTANCE         STATUS      PID
+    demo:router      RUNNING     55
 
-Expect to see the .snap file and the .xlog file. Then:
+Connect to an instance:
+
+..  code-block:: console
+
+    $ tt connect demo:router
+       • Connecting to the instance...
+       • Connected to /var/run/tarantool/demo/router/router.control
+
+    /var/run/tarantool/demo/router/router.control>
+
+Stop a specific instance:
+
+..  code-block:: console
+
+    $ tt stop demo:storage1
+       • The Instance demo:storage1 (PID = 56) has been terminated.
+
+Stop all running instances of the ``demo`` application:
+
+..  code-block:: console
+
+    $ tt stop demo
+       • The Instance demo:router (PID = 55) has been terminated.
+       • can't "stat" the PID file. Error: "stat /var/run/tarantool/demo/storage1/storage1.pid: no such file or directory"
+       • The Instance demo:storage2 (PID = 57) has been terminated.
+
+.. note::
+
+    The error message indicates that ``storage1`` is already not running.
+
+.. _admin-start_stop_instance-running_locally:
+
+Running Tarantool locally
+-------------------------
+
+Sometimes you may need to run a Tarantool instance locally, for example, for test
+purposes. ``tt`` runs in a local environment if it finds a ``tt.yaml`` configuration
+file in the current directory or any of its enclosing directories.
+To force ``tt`` into the local mode, add the ``-L`` or ``--local`` argument.
```

Comment:
Let's add a link to https://www.tarantool.io/en/doc/latest/reference/tooling/tt_cli/configuration/#launch-modes somewhere. And I didn't quite understand which `tt` command should be run with the `-L` flag.

Reply:
moved the part about --local in the end of the section with a separate example.